build.sh

```diff
@@ -12,9 +12,9 @@ fi
 if ! grep -q "kosist.test" "/etc/hosts"; then
     echo "Adding kosist.test to hosts file...";nano
 
-    if [[ "$platform" == "win" ]]; then
-        echo '127.0.0.1 kosist.test' >> /etc/hosts || echo "Failed to add kosist.test to hosts file";
-    else
+    if [[ "$platform" != "win" ]]; then
+    #     echo '127.0.0.1 kosist.test' >> /etc/hosts || echo "Failed to add kosist.test to hosts file"; Doesn't work for windows
```

Comment:
Te kaut kāds konkrēts error krita ārā? Cik pats mēģināju, tad strādāja, varbūt var atstāt, ja vien netiek bloķēta visa pārējā plūsma - idejiski vajadzētu būt tā, ka izprintējas tas "Failed to add..." un scripts turpina darboties.

Reply:
Man atverās Nano terminālī un bash.sh neizpildās rezultātā. Šis solis šķiet vairāk kosmētisks pēc dabas, pats izmantoju localhost, lai redzētu vietni. Vari iedot piemēru ko domā tad tur likt?

Reply:
Ok, tad lai pagaidām paliek kā ir, domāju, ka nebūs tik būtiski